src/Components/Footer/Footerlink/index.css

```diff
@@ -0,0 +1,8 @@
+.Footerlink{
```

Comment:
In this case, to respect SUIT Convention with Pascal Case, you could use `FooterLink`, instead of `Footerlink`

Reply:
doing it